QUICKSTART.md

```diff
@@ -168,37 +159,32 @@ The Swiss Army Kube repository has three main directories that provide a minimal
 
 * `charts`  - local Helm repository for Helm charts that can't be retrieved from public repositories.
 * `docs`    - more detailed documentation and various FAQ's
-* `example` - directory to be used as a template for your projects that includes configuration files for modules and variables.
+* `examples` - directories with different types of SAK use-cases to be used as a template for your projects that includes configuration files for modules and variables.
 * `modules` - Terraform modules (must-have and optional) to deploy your cluster with.
 
 ### Project Structure (Example Directory) 
 
-The `swiss-army-kube/example` directory is a boilerplate for your projects. Use it as a template. You can rename it to your project name for convenience. Make as many projects as you need by cloning and modifying this directory.  
+The `swiss-army-kube/examples` directory are a boilerplates for your projects, use them as a template. You can rename it to your project name for convenience. Make as many projects as you need by cloning and modifying this directory.
```

Comment:
The `swiss-army-kube/examples` directory contains project examples that you can use as boilerplates to start your new projects. Pick one, rename it to your project name for convenience, and modify the directory as required. This way you can create as many projects as you need really fast.